components/template-cs-style-rules.tpl

```diff
@@ -136,6 +136,11 @@ body {
   color: var(--content-button-hover-color);
```

Comment:
Designer comment: button padding-top: 5px, button text should be black and button text should be aligned middle